### PR TITLE
Adding jglick to ant

### PR DESCRIPTION
**Pull request by jglick**
# Description

Would like to be able to release https://github.com/jenkinsci/ant-plugin/pull/33 but especially https://github.com/jenkinsci/ant-plugin/pull/35 which is important for PCT.

Currently lists only @armfergom, though I was the last to cut a release and @Wadeck before me. Also CC @batmat, not sure if anyone else wants to be listed here.

(If granted, also make me a repo admin please so I can set up Release Drafter.)

# Submitter checklist for changing permissions

### Always

- [X] Add link to plugin/component Git repository in description above

### When adding new uploaders (this includes newly created permissions files)

- [X] [Make sure to `@`mention an existing maintainer to confirm the permissions request, if applicable](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)
- [X] Use the Jenkins community (LDAP) account name in the YAML file, not the GitHub account name
- [X] [All newly added users have logged in to Artifactory at least once](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)

#### Merge permission to GitHub repository
- [ ] Check this if newly added person also needs to be given merge permission to the GitHub repo.

### CODE REVIEW

**Comment by slide** [comment]
:+1: from me

**Comment by slide** [comment]
Close/reopen for build

**Comment by oleg-nenashev** [comment]
Still needs authorization according to the Jenkins process

File: permissions/plugin-ant.yml [hunk 1]
```diff
@@ -4,4 +4,5 @@ github: "jenkinsci/ant-plugin"
 paths:
 - "org/jenkins-ci/plugins/ant"
 developers:
-- "armfergom"
\ No newline at end of file
+- "armfergom"
+- "jglick"
```
Comment (fcojfernandez) [inline permissions/plugin-ant.yml]: @jglick I would be interested in be listed as developer 
```suggestion
- "jglick"
- "fcojfernandez"
```

**Comment by oleg-nenashev** [comment]
There is a response timeout in https://groups.google.com/forum/#!msg/jenkinsci-dev/sl9eTjIA8F0/n5XkUFmoAwAJ . I will proceed with the merge

**Pull request merged**